### PR TITLE
Update versions of communication for release of networktraversal

### PR DESCRIPTION
**Pull request by AriZavala2**

### CODE REVIEW

File: sdk/communication/azure-communication-chat/pom.xml [hunk 1]
```diff
@@ -50,12 +50,12 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-communication-common</artifactId>
-      <version>1.0.4</version> <!-- {x-version-update;com.azure:azure-communication-common;current} -->
+      <version>1.1.0-beta.1</version> <!-- {x-version-update;com.azure:azure-communication-common;current} -->
```
Comment (AriZavala2) [inline sdk/communication/azure-communication-chat/pom.xml:53]: For network traversal this is 1.0.4. Shouldn't this also be 1.0.4 here? @g2vinay

Comment (g2vinay) [reply sdk/communication/azure-communication-chat/pom.xml:53]: yes, we should update the tag to dependency here, if its supposed to depend on last released version.

File: sdk/communication/azure-communication-common/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,15 @@
 # Release History
 
+## 1.1.0-beta.1 (Unreleased)
```
Comment (g2vinay) [inline sdk/communication/azure-communication-common/CHANGELOG.md:3]: This changelog change should be added to CHANGELOG files in other communication packages too.

File: sdk/communication/azure-communication-chat/CHANGELOG.md [hunk 3]
```diff
@@ -1,9 +1,17 @@
 # Release History
 
-## 1.1.0-beta.3 (2021-08-30)
+## 1.1.0-beta.3 (Unreleased)
 ### Dependency updates
```
Comment (g2vinay) [inline sdk/communication/azure-communication-chat/CHANGELOG.md:4]: this should move under Other changes section.

File: sdk/communication/azure-communication-callingserver/CHANGELOG.md [hunk 4]
```diff
@@ -1,14 +1,22 @@
...
 
-## 1.0.0-beta.6 (Unreleased)
+## 1.0.0-beta.5 (Unreleased)
+
 ### Features Added
 - Add support for TokenCredential authentication with CallingServerClientBuilder.
 - Added support for custom options(Recordingcontenttype, Recordingchanneltype, Recordingformattype) for Mixed Audio feature
 
-## 1.0.0-beta.5 (2021-08-30)
 ### Dependency updates
```
Comment (g2vinay) [inline sdk/communication/azure-communication-callingserver/CHANGELOG.md:9]: this should come under Other changes section.

File: sdk/communication/azure-communication-identity/CHANGELOG.md [hunk 5]
```diff
@@ -2,6 +2,17 @@
 
 ## 1.2.0-beta.1 (Unreleased)
 
+### Dependency updates
```
Comment (g2vinay) [inline sdk/communication/azure-communication-identity/CHANGELOG.md:5]: Move this under other changes section 
 
 as #### Dependency updates

File: sdk/communication/azure-communication-phonenumbers/CHANGELOG.md [hunk 6]
```diff
@@ -1,9 +1,18 @@
 # Release History
 
-## 1.0.4 (2021-08-30)
+## 1.1.0-beta.1 (Unreleased)
+
 ### Dependency updates
```
Comment (g2vinay) [inline sdk/communication/azure-communication-phonenumbers/CHANGELOG.md:5]: move under Other changes section as `#### Dependency updates`

File: sdk/communication/azure-communication-sms/CHANGELOG.md [hunk 7]
```diff
@@ -1,9 +1,18 @@
 # Release History
 
-## 1.0.4 (2021-08-30)
+## 1.1.0-beta.1 (Unreleased)
+
 ### Dependency updates
```
Comment (g2vinay) [inline sdk/communication/azure-communication-sms/CHANGELOG.md:5]: move under Other changes section as #### Dependency updates

**Review by g2vinay — APPROVED** [review]

**Review by g2vinay — APPROVED** [review]

**Pull request merged**